js/src/components/google-combo-account-card/connect-ads/connect-existing-account.js

```diff
@@ -58,17 +57,16 @@ const ConnectExistingAccount = ( { onCreateClick } ) => {
 			return;
 		}
 
-		clearDetailedErrorBySlot( ERROR_SLOT );
+		clearDetailedErrorBySlot( ERROR_SLOTS.GOOGLE_ADS_CONNECTION );
 
 		setLoading( true );
 		try {
 			await connectGoogleAdsAccount();
 			await fetchGoogleAdsAccountStatus();
 			await refetchGoogleAdsAccount();
-		} catch ( error ) {
-			receiveDetailedError( ERROR_SLOT, {
-				...error,
-				code: error.statusText,
+		} catch ( e ) {
```

Comment:
```suggestion
		} catch ( error ) {
```
Let's keep the `error` name here instead of `e`